domains/rizqi.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Portofolio Website.",
+    "repo": "https://github.com/RizqiSeijuuro/portfolio",
+    "owner": {
+        "username": "RizqiSeijuuro",
+        "email": "rizqiansyah52@gmail.com"
+    },
+    "record": {
+        "CNAME": "RizqiSeijuuro.github.io/portfolio"
```

`CNAME` record must be lowercased and must contain only a domain name
```suggestion
        "CNAME": "rizqiseijuuro.github.io"
```